packages/client/src/pages/EmissionsMetricsPage/EmissionsMetricsPage.tsx

```diff
@@ -20,6 +20,9 @@ import useStyles from './emissionsMetricsStyles'
 import EmissionsSidePanel from './EmissionsSidePanel/EmissionsSidePanel'
 import { EmissionsFilters } from './EmissionsFilterBar/utils/EmissionsFilters'
 import { EstimationResult } from '@cloud-carbon-footprint/common'
+import {ThemeProvider} from "styled-components";
+
+const BASE_URL = "/api";
```

Comment:
in backstage plugins, you are supposed to rely on their built-in discovery mechanisms to find the API you're trying to talk to. see related changes in the remove service hooks.

there are different ways to slice this but the simplest is simply to inject the correct baseUrl. in our current hack, baseUrl is equal to `localhost:7007/api/cloud-carbon-footprint` which we retrieve from a [discovery mechanism](https://github.com/lcoffeejohnson/backstage/blob/cloud-carbon-footprint-plugin/plugins/cloud-carbon-footprint/src/components/ExampleComponent/ExampleComponent.tsx#L78)